packages/wonder-stuff-i18n/src/bin/gen-potfile.js

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env node
+// @flow
+/* istanbul ignore file */
+/**
+ * Generates a POT file containing all strings marked for translation. The POT
+ * file is output to STDOUT.
+ *
+ * You can specify files on the command-line and this script will only extract
+ * strings from those files, instead of from all JavaScript file.
+ *
+ * USAGE:
+ *
+ *   # Extract strings from all JS files and export to a shared POT file
+ *   ./gen-potfile.js javascript/**\/*.{js,jsx} > JavaScript.pot
+ *
+ *   # Extract strings from a specific file and output to STDOUT
+ *   ./gen-potfile.js javascript/about/about.jsx
```

Comment:
nit: update usage to show how this would be used when installed as an npm package.

Reply:
Done!